subsys/bluetooth/controller/Kconfig

```diff
@@ -945,7 +945,7 @@ config BT_CTLR_CONN_ISO_SDU_LEN_MAX
 	int "Maximum Connected Isochronous Channel SDU Length"
 	depends on BT_CTLR_CONN_ISO
 	range 1 4095
-	default 251
+	default BT_ISO_TX_MTU
```

Comment:
@cvinayak is BT_CTLR_CONN_ISO_SDU_LEN_MAX including the size of the `bt_hci_iso_sdu_hdr`/`bt_hci_iso_sdu_ts_hdr` headers?

Reply:
No, this is SDU length (no HCI layer)